backend/app/schemas/llm_service.py

```
@@ -0,0 +1,117 @@
+
+    id: str = Field(..., description="Completion ID")
+    object: str = Field(default="chat.completion", description="Object type")
+    created: int = Field(..., description="Creation timestamp")
+    model: str = Field(..., description="Model used")
+    choices: list[CompletionChoice] = Field(
+        ..., description="List of completion choices"
+    )
+    usage: CompletionUsage | None = Field(None, description="Token usage information")
+
+
+class EmbeddingRequest(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)